SECURITY.md

```diff
@@ -0,0 +1,174 @@
+slsa-github-generator project and will not be disseminated to other projects
+unless it is necessary to get the issue fixed.
+
+As the security issue moves from triage, to identified fix, to release planning
+we will keep the reporter updated.
+
+## Security Release & Disclosure Process
+
+Security vulnerabilities should be handled quickly and sometimes privately. The
+primary goal of this process is to reduce the total time users are vulnerable
+to publicly known exploits.
+
+### Private Disclosure
+
+We ask that all suspected vulnerabilities be privately and responsibly
+disclosed via the [private disclosure process](#reporting-a-vulnerability)
+outlined above.
+
+Fixes may be developed and tested by the [Security Team](#security-team) in
+private repositories if deemed necessary.
+
+### Public Disclosure
+
+Vulnerabilities are disclosed publicly as GitHub
+[Security Advisories](https://github.com/slsa-framework/slsa-github-generator/security/advisories).
+
+A public disclosure date is negotiated by the [Security Team](#security-team)
+and the bug submitter. We prefer to fully disclose the bug as soon as possible
+once a user mitigation is available. It is reasonable to delay disclosure when
+the bug or the fix is not yet fully understood, the solution is not
+well-tested, or for vendor coordination. The timeframe for disclosure is from
+immediate (especially if it's already publicly known) to several weeks. For a
+vulnerability with a straightforward mitigation, we expect report date to
+disclosure date to be on the order of 14 days.
+
+If you know of a publicly disclosed security vulnerability please IMMEDIATELY
+[report the vulnerability](#reporting-a-vulnerability) to inform the
+[Security Team](#security-team) about the vulnerability so they may start the
+patch, release, and communication process.
+
+If possible the Security Team will ask the person making the public report if
+the issue can be handled via a private disclosure process. If the reporter
+denies the request, the Security Team will move swiftly with the fix and
+release process. In extreme cases you can ask GitHub to delete the issue but
+this generally isn't necessary and is unlikely to make a public disclosure less
+damaging.
+
+### Security Releases
+
+Once a fix is available it will be released and announced via the project on
+GitHub. Security releases will announced and clearly marked as a security
+release and include information on which vulnerabilities were fixed. As much as
+possible this announcement should be actionable, and include any mitigating
+steps users can take prior to upgrading to a fixed version.
+
+Fixes will be applied in patch releases to all [supported
+versions](#supported-versions).
+
+### Severity
+
+The [Security Team](#security-team) evaluates vulnerability severity on a
+case-by-case basis, guided by [CVSS 3.1](https://www.first.org/cvss/v3.1/specification-document).
+
+## Security Posture
```

Comment:
There is currently nothing here about how we manage repo owners/admins and settings. I'm not sure it needs to be in the initial version but given that folks are referencing and executing code directly from the repo it's worth writing down at some point.

Maybe as part of writing a formal contributor ladder/onboarding process that we mostly spitballed for Naveen.
https://github.com/slsa-framework/slsa-github-generator/pull/428#issuecomment-1194931713

Reply:
Agree that it would be very useful to capture this, let's file an issue to ensure we don't forget?

Reply:
https://github.com/slsa-framework/slsa-github-generator/issues/1291